From 732ff5861a1525944a927439d1c075ac269788ce Mon Sep 17 00:00:00 2001
From: Marian Csontos <mcsontos@redhat.com>
Date: Thu, 27 Jul 2023 11:37:01 +0200
Subject: [PATCH] Tweak cargo.toml to work with vendor directory

Mock works offline, cargo would try to download the files from github.
So cargo vendor has to be run first, and then change the Cargo.toml to
make mock happy.
---
 Cargo.toml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index c2b496ac..562c40d3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,6 +11,7 @@ anyhow = "1.0"
 base64 = "0.21"
 byteorder = "1.4"
 clap = { version = "4.3", default-features = false, features = ["std", "help", "usage", "error-context", "suggestions"] }
+#crc32c = { git = "https://github.com/zowens/crc32c", branch = "master" }
 crc32c = "0.6"
 data-encoding = "2.4"
 exitcode = "1.1.2"
@@ -27,7 +28,8 @@ quick-xml = "0.29"
 rand = "0.8"
 rangemap = "1.3"
 roaring = "0.10"
-rio = { git = "https://github.com/jthornber/rio", branch = "master", optional = true }
+#rio = { git = "https://github.com/jthornber/rio", branch = "master", optional = true }
+rio = { version = "0.9.4", optional = true }
 safemem = "0.3"
 threadpool = "1.8"
 thiserror = "1.0"
-- 
2.41.0

